src/Tracking/TrackingData/DonationFormsData.php

```diff
@@ -81,6 +83,7 @@ protected function getData()
                 'form_template' => $this->getFormTemplate($formId),
                 'donor_count' => $this->formDonorCounts[$formId],
                 'revenue' => $this->formRevenues[$formId],
+                'event_block' => FormBlocks::formId($formId)->hasBlock('give_event_block'),
```

Comment:
Would it be `givewp/event-tickets` here?

Reply:
Ah, copy/paste error. I'll fix that.

Reply:
Resolved, see https://github.com/impress-org/givewp/pull/7341/commits/96706297af9d132094a7b8ea287a08886365b026.